app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/ChangePasscodeFragment.kt

```diff
@@ -104,4 +103,9 @@ class ChangePasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>(
             }
         }
     }
+
+    override fun onStop() {
+        super.onStop()
+        binding.input.hideSoftKeyboard()
```

Comment:
Doing this in onStop() has turned out to be a problem.

Reply:
missed one cherry :)